src/cohort.js

```diff
@@ -0,0 +1,44 @@
+export default class Cohort {
+  constructor(name, capacity = 24) {
+    if (typeof name !== 'string') {
+      throw new Error('name input must be String')
+    }
+
+    if (name.length < 1) {
+      throw new Error('name input must not be blank')
+    }
+
+    if (typeof capacity !== 'number') {
+      throw new Error('capacity input must be Number')
+    }
+
+    this.name = name
+    this.capacity = capacity
+    this.id = Cohort.nextId()
```

Comment:
I noticed you are finding and removing the cohorts by name. Curious what this id is for?

Reply:
Probably not anything I couldn't use name for. Just put it in by habit really.